megamek/src/megamek/common/AmmoType.java

```diff
@@ -13457,12 +13464,14 @@ public AmmoType createMunitionType(AmmoType base) {
                     || (munition.getAmmoType() == AmmoType.T_MML) || (munition.getAmmoType() == AmmoType.T_NLRM))
                     && (munition.getMunitionType() == AmmoType.M_SEMIGUIDED)) {
                 cost *= 3;
+                bv *= 1;
```

Comment:
multiplying by 1 doesn't do anything?

Reply:
Reminder that the BV has been checked. Beats have a blank line with no idea that it's been compared to source.